Nitrox.BuildTool/Program.cs

```diff
@@ -86,7 +88,7 @@ static bool ValidateUnityGame(GameInstallData game, out string error)
                     throw new Exception(error);
                 }
             }
-
+            
             game ??= new GameInstallData(NitroxUser.GamePath);
```

Comment:
NitroxModel calls Log functions when finding your install. While Log may not be directly used by Nitrox.BuildTool, it needs to be set up in BuildTool so that logging in NitroxModel works inside BuildTool.